notebooks/cuml_benchmarks.ipynb

```diff
@@ -293,8 +147,8 @@
    "source": [
     "algorithm = \"KNeighborsRegressor\"\n",
```

Comment:
Is `algorithm` used? maybe it should be `execute_benchmark(algorithm)` in the final line?

Reply:
I removed algorithm everywhere. I must have missed this one.

Reply:
At least I thought I removed algorithm everywhere. I intended to, anyways. It's removed now for sure.